cassandra-analytics-core/src/test/java/org/apache/cassandra/spark/bulkwriter/MockBulkWriterContext.java

```diff
@@ -77,6 +78,23 @@ public class MockBulkWriterContext implements BulkWriterContext, ClusterInfo, Jo
     private int sstableDataSizeInMB = 128;
     private int sstableWriteBatchSize = 2;
 
+    private final Map<String, String> jobStats = new HashMap<>();
+
+    public Map<String, String> jobStats()
+    {
+        return jobStats;
+    }
```

Comment:
NIT: missing line break
```suggestion
    }

```